<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="container">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <widget class="ScribbleArea" name="widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openImage"/>
   <addaction name="saveImage"/>
   <addaction name="cleanImage"/>
   <addaction name="separator"/>
   <addaction name="acceptTympanum"/>
   <addaction name="cleanTympanum"/>
   <addaction name="separator"/>
   <addaction name="acceptPunction"/>
   <addaction name="cleanPunction"/>
  </widget>
  <action name="openImage">
   <property name="text">
    <string>AbrirImagen</string>
   </property>
   <property name="toolTip">
    <string>Abrir Imagen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="cleanImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Limpiar Imagen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="acceptTympanum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Aceptar Tímpano</string>
   </property>
  </action>
  <action name="cleanTympanum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Limpiar Tímpano</string>
   </property>
  </action>
  <action name="saveImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar Imagen</string>
   </property>
  </action>
  <action name="acceptPunction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Aceptar Perforación</string>
   </property>
  </action>
  <action name="cleanPunction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Limpiar Perforación</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScribbleArea</class>
   <extends>QWidget</extends>
   <header>scribblearea.h</header>
   <container>1</container>
   <slots>
    <slot>saveImage()</slot>
    <slot>clearImage()</slot>
    <slot>resetAirdrum()</slot>
    <slot>resetPunction()</slot>
    <slot>acceptAirdrum()</slot>
    <slot>acceptPunction()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>saveImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptTympanum</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>acceptAirdrum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptPunction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>acceptPunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleanImage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cleanImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>acceptAirdrum()</slot>
  <slot>acceptPunction()</slot>
  <slot>saveFile()</slot>
  <slot>cleanImage()</slot>
 </slots>
</ui>
